Import.hs

```diff
@@ -35,3 +37,26 @@ haddockUrl sname pkgver name = HaddockR sname
   where
     toDash '.' = '-'
     toDash c = c
+
+track
+    :: MonadIO m
+    => String -> m a -> m a
+track name inner = do
```

Comment:
I think in future a more reliable way to get time would be via `Criterion.Measurement`:

http://hackage.haskell.org/package/criterion-1.1.1.0/docs/Criterion-Measurement.html

E.g.
``` haskell
do initializeTime
   start <- getCPUTime
   r <- inner
   end <- getCPUTime
```